modules/status.py

```diff
@@ -117,58 +142,62 @@ def main(bot, trigger, options):
-                               config.get('zppixbot_status', 'password'))
+                    site.login(bot.config.status.wiki_username,
+                               bot.config.status.wiki_password))
                 except errors.LoginError as e:
                     print(e)
                     raise ValueError("Login failed.")
                 except requests.exceptions.Timeout:
-                    bot.say(trigger.nick + ": We're experinecing delays "
-                            + "connecting to that wiki. Try again in a few "
-                            + "minutes. If this continues, let us know in "
-                            + "#ZppixBot.", trigger.sender)
+                    bot.reply("We're experinecing delays "
+                            + "connecting to that wiki. Try again in a few minutes.")
+                    if bot.config.status.support_channel is not None:
+                        bot.say("If this continues, let us know in "
+                            + "{}".format(bot.config.status.support_channel))
                 except requests.exceptions.TooManyRedirects as e:
-                    bot.say(trigger.nick + ": We couldn't connect to that "
-                            + "wiki. I've alerted a maintainer in #ZppixBot.",
-                            trigger.sender)
-                    bot.say("Redirect Error: " + e, '#ZppixBot-logs')
+                    bot.reply("We couldn't connect to that wiki.")
+                    if bot.config.status.support_channel is not None:
+                        bot.say("I've alerted a maintainer in {}"
+                            + "{}".format(bot.config.status.support_channel))
+                    print(e)
+                    raise ValueError("Redirect error")
                 except requests.exceptions.ConnectionError as e:
-                    bot.say(trigger.nick + ": We couldn't connect to that "
-                            + "wiki. I've alerted a maintainer in #ZppixBot.",
-                            trigger.sender)
-                    bot.say("ConnectionError: " + e, '#ZppixBot-logs')
+                    bot.reply("We couldn't connect to that wiki.")
+                    if bot.config.status.support_channel is not None:
+                        bot.say("I've alerted a maintainer in {}"
+                            + "{}".format(bot.config.status.support_channel))
+                    raise ValueError("Connection error")
                 except requests.exceptions.RequestException as e:
-                    bot.say(trigger.nick + ": A fatal error occured. I've "
-                            + "alerted a maintainer in #ZppixBot.",
-                            trigger.sender)
-                    bot.say("Fatal Error: " + e, '#ZppixBot-logs')
+                    bot.reply("A fatal error occured.")
+                    if bot.config.status.support_channel is not None:
+                        bot.say("I've alerted a maintainer in {}"
```

Comment:
```suggestion
                        bot.say("I've alerted a maintainer in "
```
Like other lines similar to this, the following line already includes the brackets.